### ISSUE TITLE
"Ubuntu" still present on homepage

### ISSUE THREAD

**Issue by paulproteus** [openhatch/oh-greenhouse#7]
There are a few places the string "Ubuntu" still appears, even though it's for Debian now.

On http://greenhouse-daveeloo.dotcloud.com/ :   
- "Potential Ubuntu contributors"
- "Ubuntu.com" and other ubuntu-related links in header

On contributor pages , e.g. http://greenhouse-daveeloo.dotcloud.com/contributors/EMAIL_OMITTED :
- "Ubuntu Developer: No"

**Comment by daveeloo** [comment]
Also removed trademark and changed favicon. Let me know if removing the trademark is a bad idea (it's an Ubuntu trademark).

**Comment by daveeloo** [comment]
Added copyright as Debian Project (let me know if that's a bad idea)

**Comment by paultag** [comment]
ISTR a discussion that seemed to point at this sort of "copyright assignment" as being basically meaningless; but I can't recall exactly.

**Comment by andrewsomething** [comment]
@paultag I think you mean this thread: https://lists.debian.org/debian-project/2013/02/msg00035.html

**Comment by paultag** [comment]
Quite! Thanks, @andrewsomething !

**Comment by paulproteus** [comment]
@daveeloo FWIW, speaking from my understanding of copyright law in the US, the copyright is joint between you (for your contributions) and the individual people who created the rest of the code, which basically is @andrewsomething .

**Comment by andrewsomething** [comment]
When the project started, I didn't foresee it having much use for anyone else. I kept meaning to get around to adding appropriate copyright headers, but obviously didn't. Lesson learned. While I'm the sole author of most of it, Daniel Holbach should also be a copyright holder of `uploads/management/commands/get-udd-data.py` and while I wrote it (and should receive all blame) we've tried to stick to requiring a review before commits to trunk. So most of the code has been written by me in a feature branch and then merged to trunk by Daniel. I don't know if his work was on the clock for Canonical or not though. I can get further clarification if needed.
